bodhi/client/__init__.py

```diff
@@ -244,13 +246,24 @@ def _save_override(url, user, password, staging, edit=False, **kwargs):
         kwargs (dict): Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, username=user, password=password, staging=staging)
+
     resp = client.save_override(nvr=kwargs['nvr'],
                                 duration=kwargs['duration'],
                                 notes=kwargs['notes'],
                                 edit=edit,
                                 expired=kwargs.get('expire', False))
+    resp['wait'] = kwargs['wait']
     print_resp(resp, client)
 
+    if resp['wait']:
+        if 'release_id' in resp.build:
+            release = client.get_releases(ids=[resp.build.release_id])['releases'][0]
+            args = ('koji', 'wait-repo', '{}-build'.format(release.dist_tag),
+                    '--build={}'.format(resp.build.nvr))
```

Comment:
This is duplicating effort with the other code that also forms this statement. We should share code so there's only one implementation.